storage.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"os"
+
+	"cloud.google.com/go/storage"
+	"golang.org/x/net/context"
+)
+
+var (
+	client *storage.Client
+
+	bucket *storage.BucketHandle
+)
+
+func init() {
+	var err error
+	client, err = storage.NewClient(context.Background())
+	if err != nil {
+		panic(err)
+	}
+}
+
+func handleStaticRequest(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Served-With-Love-By", "Gophers")
+
+	bucket := client.Bucket(os.Getenv("SITE_BUCKET"))
```

Comment:
Maybe it might be helpful to check first if `SITE_BUCKET` is set. So, it's possible to do it in `init` function, or somewhere in the first part of `main` function.

Something like this:
```
siteEnv := os.Getenv("SITE_BUCKET")

if len(siteEnv)==0 {
 // we can't start app, log error and exit
}
```